tractor/_entry.py

```diff
@@ -57,6 +57,9 @@ def _trio_main(
 ) -> None:
     """Entry point for a `trio_run_in_process` subactor.
     """
+    # TODO: make a global func to set this or is it too hacky?
```

Comment:
I'm starting to think this might be a good thing to do in subactors to avoid users thinking they can debug with plain old `breakpoint()` (which will still engage the debugger but potentially cause all sorts of console clobbering because there is no locking around the parent tty).